samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json

```diff
@@ -0,0 +1,92 @@
+﻿{
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Conversation will be created between the selected users.",
+      "wrap": true
+    },
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Enter Name of Group Chat"
+    },
+    {
+      "type": "Input.Text",
+      "spacing": "Medium",
+      "id": "title"
+    },
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Enter Message"
+    },
+    {
+      "type": "Input.Text",
+      "spacing": "Medium",
+      "id": "message"
+    },
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Select Context"
+    },
+    {
+      "type": "Input.ChoiceSet",
+      "choices": [
+        {
+          "title": "CurrentContext",
```

Comment:
Let's change these titles to more descriptive (keep value the same so code can stay unchanged)